### PR TITLE
Embedded content cleanup

### PR DESCRIPTION
**Pull request by pvh**
Cleans up BoardInBoard, FileInBoard, and makes Badge size way more reliable.

### CODE REVIEW

File: src/renderer/components/content-types/TextContent.tsx [hunk 1]
```diff
@@ -50,7 +50,7 @@ export default function TextContent(props: Props) {
 
   return (
     <div
-      className="TextContent"
+      className="TextContent BoardCard--standard"
```
Comment (mjtognetti) [inline src/renderer/components/content-types/TextContent.tsx]: Seems wrong that `TextContent` has a dependency on `BoardCard.css`. An alternative could be to have a containing structure (the card) implement the `BoardCard--standard` styles and we can have `TextContent` just fill that parent without caring about the styles.

File: src/renderer/components/Heading.css [hunk 2]
```diff
@@ -5,3 +5,11 @@
   text-overflow: ellipsis;
   white-space: nowrap;
 }
+
+.Heading--wrap {
+  white-space: normal;
+  display: -webkit-box;
```
Comment (mjtognetti) [inline src/renderer/components/Heading.css]: I'm all about this

**Comment by mjtognetti** [comment]
Some visual notes:

The avatar and clipboard icon are different sizes (not sure if this introduced):
Screen Shot 2019-11-12 at 12 25 47 PM

The default board-in-board and file-in-board badges are pretty large (too large in my mind):
Screen Shot 2019-11-12 at 12 26 52 PM

File: src/renderer/components/content-types/TextContent.tsx (hunk 1 above)
Comment (pvh) [reply src/renderer/components/content-types/TextContent.tsx]: Filing a follow-up.

**Pull request merged**